<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kOnlineTransferFormDecl</class>
 <widget class="QDialog" name="kOnlineTransferFormDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transfer Form</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Order Account</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="KComboBox" name="orderAccount"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Account Balance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="kMyMoneyEdit" name="orderAccountBalance">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="calculatorButtonVisibility" stdset="0">
         <bool>false</bool>
        </property>
        <property name="resetButtonVisibility" stdset="0">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Credit Transfer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioTransferNational">
          <property name="text">
           <string>National Credit Transfer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioTransferSepa">
          <property name="text">
           <string>SEPA Credit Transfer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="creditTransferEdits">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="accountUnsupported">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>54</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>198</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QGraphicsView" name="graphicsView">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="interactive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This account does not support online banking.&lt;/p&gt;&lt;p&gt;Please select another account.&lt;/p&gt;&lt;p&gt;If you think this is an error, check the plugin settings.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>188</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>53</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="sepaCreditTransferEdit" name="sepaPage"/>
        <widget class="germanCreditTransferEdit" name="germanPage"/>
       </widget>
      </item>
     </layout>
     <zorder>creditTransferEdits</zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="buttonEnque">
       <property name="text">
        <string>Enque</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="buttonSend">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
       <property name="icon">
        <iconset theme="mail-send">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="buttonAbort">
       <property name="text">
        <string>Abort</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-cancel">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>kMyMoneyEdit</class>
   <extends>QFrame</extends>
   <header>kmymoneyedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>sepaCreditTransferEdit</class>
   <extends>QWidget</extends>
   <header>onlinejobedit/sepacredittransferedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>germanCreditTransferEdit</class>
   <extends>QWidget</extends>
   <header>onlinejobedit/germancredittransferedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>orderAccount</tabstop>
  <tabstop>radioTransferNational</tabstop>
  <tabstop>radioTransferSepa</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>buttonSend</tabstop>
  <tabstop>buttonEnque</tabstop>
  <tabstop>buttonAbort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
